config/application.rb

```diff
@@ -136,6 +137,12 @@ class Application < Rails::Application
     # Add apidoc hash in headers for smarter caching
     config.middleware.use "Apipie::Middleware::ChecksumInHeaders"
 
+    Foreman::Logging.configure(:log_directory => "#{Rails.root}/log", :environment => Rails.env)
+    config.colorize_logging = Foreman::Logging.config[:colorize]
```

Comment:
Note that we also set colorize_logging in all three config/environments/* files based on SETTINGS.  Is that obsolete?